python/tidy/servo_tidy/tidy.py

```diff
@@ -389,6 +389,8 @@ def check_rust(file_name, lines):
             # No benefit over using &str
             (r": &String", "use &str instead of &String", no_filter),
             (r"^&&", "operators should go at the end of the first line", no_filter),
+            (r"\{[A-Za-z0-9]+\};", "use statement contains parenthesis for single import",
```

Comment:
One last thing and this will be good to go. The regex should also take underscores into account (we could import something like `foo_bar`).

I'm sorry that I didn't mention this before :/